configure.ac

```diff
@@ -1,4 +1,4 @@
-AC_INIT([lldpad], [1.2.99], [lldp-devel@open-lldp.org])
+AC_INIT([lldpad], [1.1.1], [lldp-devel@open-lldp.org])
```

Comment:
Why do you think this should change?

This value will be a hint on bug reports.  IE: if someone writes a bug and shows 1.2.90, then we know it is 
against latest main branch.  OTOH, 1.1.1 should be found on branch-1.1 and that will be a very specific version.